opencti-platform/opencti-front/src/private/components/common/files/CustomFileUploader.tsx

```diff
@@ -121,6 +124,25 @@ const CustomFileUploader: FunctionComponent<CustomFileUploadProps> = ({
     }
   };
 
+  function returnDisabledStatus(disabledStatus: boolean): boolean {
```

Comment:
Thanks, just for information you could have make a second `useEffect` with only `disabled` as dependency to avoid triggering the disable part when it is `formikErrors` that changes